src/app/signing.service.ts

```diff
@@ -21,20 +22,72 @@ export class SigningService {
     return jsonwebtoken.sign({ }, encodedPrivateKey, { algorithm: 'ES256', expiresIn: 60 * 10 });
   }
 
-  decryptMessages(seedHex: string, encryptedHexes: string[]): { [key: string]: any } {
+  encryptMessage(seedHex: string, recipientPubkeyBase58Check: string, message: string): string {
```

Comment:
I think it's fine to just call this parameter `recipientPublicKey`. Any string representation of a public key will necessarily be base58